### DISCUSSION TITLE
Multi level nesting for reactive properties does not work

### DISCUSSION THREAD

**Discussion by gehrisandro** [livewire/livewire#6125] [7. Bugs (archived)]
### Livewire version

v3.0.0-beta.5

### Which PHP version are you using?

PHP 8.2

### Repository URL

https://github.com/gehrisandro/livewire-nested-reactive/tree/main/app

### Steps To Reproduce

All the steps already done in this repository: https://github.com/gehrisandro/livewire-nested-reactive/tree/main/app

1. Create three components: Main, Child and SubChild
2. Create a public property on the Main component
3. Create the same public property on the Child and SubChild component and add the `Reactive` attribute
4. Create a blade file for the Main component which renders the property value and the Child component
5. Create a blade file for the Child component which renders the property value and the SubChild component
6. Create a blade file for the SubChild component which renders the property value
7. Add a button to the Main component which updates the property
8. Check the output: The property has only been updated on the Main and Child component.

### Additional information

Not sure if this is a bug or the intended behaviour.

Nesting reactive properties only work on the first level (parent / child), but updates are not passed to a child of the child.

In my demo repository there are three components:
- Main
- Child
- SubChild

The reactive prop `$value` is properly passed to the `Child` and `SubChild` component on the initial render. But if the `Main` components updates the `$value` prop only the `Child` component updates, but not the `SubChild`.

As I said before, this is may the intended behaviour but from the docs it wasn't clear for me, that this will not work.

If you need help to make this possible or to update the docs, please let me know.

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment]
@gehrisandro thanks for reporting! I'm not sure if it is a bug or not originally intended, but I think it should work. Any chance you could submit a failing test PR that demonstrates the issue so we can investigate? Thanks!

**Reply by gehrisandro** [reply]
@joshhanley Thanks for your reply. Yes, I can do this, but probably not today.
